streams/dshr_strdata_mod.F90

```diff
@@ -138,6 +138,7 @@ module dshr_strdata_mod
      real(r8)                       :: lambm0 = SHR_ORB_UNDEF_REAL     ! cosz t-interp info
      real(r8)                       :: obliqr = SHR_ORB_UNDEF_REAL     ! cosz t-interp info
      real(r8), allocatable          :: tavCoszen(:)                    ! cosz t-interp data
+     logical                        :: is_restart = .false.
```

Comment:
@mvertens I may be missing something, but I'm not seeing the logic of how is_restart gets set to true on restart? Did you mean for this to start out as true?

Reply:
You could add a comment to this to clarify how it's intended to function as well.

Reply:
@ekluzek - thanks for catching this. I think I used the is_restart flag and then never removed it. So at this point always false - and the original logic is used.  I would be tempted to just remove this for now and get this PR merged. If I turn the restart flag back I think it will require some more careful testing. Are you okay with that?

Reply:
Yes, just removing it would be fine. And I agree that's the best thing to do now, as otherwise you are right you'd need to do more testing. So you might as well remove it and go back to the previous code. Potentially this was a useful addition to make it more clear what happened on restart, but it's not required and there's not a compelling need for it.